### PR TITLE
fix: simple functional tests for blob ingestion

### PR DESCRIPTION
**Pull request by pauldambra**
adding recorded ingestion functional tests was causing other functional tests to fail

now, it doesn't

### CODE REVIEW

File: plugin-server/functional_tests/session-recordings-blob-ingestion.test.ts [hunk 1]
```diff
@@ -10,31 +9,16 @@ import { UUIDT } from '../src/utils/utils'
 import { capture, createOrganization, createTeam } from './api'
 import { waitForExpect } from './expectations'
 
-let kafka: Kafka
+// let kafka: Kafka
```
Comment (benjackwhite) [inline plugin-server/functional_tests/session-recordings-blob-ingestion.test.ts:12]: Do we need this?

**Review by benjackwhite — APPROVED** [review]
Looks good with a ignorable nit

File: plugin-server/functional_tests/session-recordings-blob-ingestion.test.ts (hunk 1 above)
Comment (pauldambra) [inline plugin-server/functional_tests/session-recordings-blob-ingestion.test.ts:12]:
```suggestion
```

Comment (pauldambra) [reply plugin-server/functional_tests/session-recordings-blob-ingestion.test.ts]: nuked it

**Pull request merged**